### PR TITLE
Minor changes to clarify pre-requisites and config for new users

### PR DESCRIPTION
**Pull request by danielpunkass**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ This is the open-source portion of the back-end, website scraping software that
...
-1.  Create a `.env` file with the following:
+1.  Create a [`.env`](https://www.npmjs.com/package/dotenv) file in the root directory of the project with the following:
 
         DEVELOPMENT=true
         CHROMEPATH="path/to/chromium/that/you/downloaded"
         # e.g. /Applications/Chromium.app/Contents/MacOS/Chromium
         PROPRIETARY_SITE_SCRAPERS_PATH="./../proprietary/site-scrapers" (optional, example)
 
+1.  Install [node.js](https://nodejs.org/en/) version 14.0 or later
 1.  Install `prettier` and `eslint`; make sure you run them before making any commits.
```
Comment (harcod) [inline README.md:20]: You could also add `npm` version 7.0 or later

**Review by harcod — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (danielpunkass) [reply README.md:20]: @harcod Ah! I only ran into a version problem with my vesrion of node, so I didn't focus on the npm requirement. Does npm not come "for free" with node?

Comment (harcod) [reply README.md:20]: I'm not sure of the linkage between `npm` and `node` as far as installation goes, but I do know that they can be updated independently.  You might want to look at the [README.md](https://github.com/livgust/macovidvaccines.com/blob/master/README.md) file in front-end which I think has a better all-around explanation.  Maybe you can borrow from that README for this one.

Comment (danielpunkass) [reply README.md:20]: Thanks, @harcod !

Comment (harcod) [reply README.md]: I like it!  Thanks for the PR.  It's very easy for those that have been working on the project to move past the README file after they get everything working.

Comment (danielpunkass) [reply README.md]: Thanks again for the quick review and for appreciating the newbie care...

**Comment by danielpunkass** [comment]
Argh. Didn't mean to commit a bunch of local hacks with this. Drafting...

**Comment by danielpunkass** [comment]
Will reopen when I can with a clean patch.

**Pull request closed without merging**